.github/workflows/js-ts-lint.yml

```diff
@@ -0,0 +1,28 @@
+name: js-ts-lint
```

Comment:
From what I can tell I can't watch different files from a single job. Maybe I missed some docs but the way this is set up it will only run on relevant changes, whereas joining them into a global job means it runs every time.

Reply:
Resolved